Tasks/ExtractFilesV1/extractfilestask.ts

```diff
@@ -20,18 +21,33 @@ tl.debug('win: ' + win);
 var xpTarLocation: string;
 var xpUnzipLocation: string;
 // 7zip
-var xpSevenZipLocation: string;
-var winSevenZipLocation: string = path.join(__dirname, '7zip/7z.exe');
+let xpSevenZipLocation: string;
+let defaultWinSevenZipLocation: string = path.join(__dirname, '7zip/7z.exe');
+let winSevenZipLocation: string;
```

Comment:
Could you please clarify what's the reason for adding this variable?

Reply:
Seems like it is an excess variable and we can avoid using it.